### DISCUSSION TITLE
How to config remote access when installing debian?

### DISCUSSION THREAD

**Discussion by mo-han** [zbm-dev/zfsbootmenu#574] [Questions] [answered]
There's no remote access related content in the debian installation section, and no debian related content in the remote access section.
How to setup mkinitcpio in debian?

**Comment by ahesford** [comment, marked as answer]
I imagine you'd have to install mkinitcpio manually and wouldn't recommend doing so. You can follow the Dracut instructions [in the guide](https://docs.zfsbootmenu.org/en/v2.3.x/guides/general/remote-access.html), or else use the [containerized build process](https://docs.zfsbootmenu.org/en/v2.3.x/guides/general/container-example.html).

**Reply by mo-han** [reply]
Thanks for the advice. Are there any more links of useful info?
